xml/System.IO/Path.xml

```diff
@@ -2310,7 +2310,9 @@ The method does not verify that the path exists or test to see if the current pr
 
 4.  The Windows directory.
 
-In .NET 7 and later, when the process calling the method is running as `SYSTEM`, the method skips the preceding sequence and returns `C:\Windows\SystemTemp`. This location is inaccessible to processes not running as `SYSTEM`.
+In .NET 7 and later versions (or .NET 6 version 6.0.32 and later), OR if your operating system is Windows 10 or Windows Server build 20348 or later and you're using any .NET Framework version, when the process calling the method is running as `SYSTEM`, the method skips the preceding sequence and returns `C:\Windows\SystemTemp`. This location is inaccessible to processes not running as `SYSTEM`.
+
+On Full .NET Framework, this behavior can be opted out by setting `COMPlus_Disable_GetTempPath2` environment variable to `true` (all other values are ignored).
```

Comment:
```suggestion
In .NET 6 version 6.0.32 and later versions or in .NET Framework, if your operating system is Windows 11 or Windows Server build 20348 or later, when the process calling the method is running as `SYSTEM`, the method skips the preceding sequence and returns `C:\Windows\SystemTemp`. This location is inaccessible to processes not running as `SYSTEM`.

On .NET Framework, this behavior can be opted out by setting the `COMPlus_Disable_GetTempPath2` environment variable to `true` (all other values are ignored).
```

Reply:
Updated.

Reply:
It should be "and later" instead of plus notation to follow conventions (like it's already used here: https://learn.microsoft.com/en-us/dotnet/api/system.environment.osversion?view=net-8.0#remarks).